doc/UserGuide.md

```diff
@@ -53,18 +53,22 @@ Examples:
 Shows a list of all persons in the address book.<br>
 Format: `list`
 
+### Sort all persons : `sort`
+Sort all persons in the address book in an alphabetical order to the user.
+Format: `sort`
+
 ### Finding all persons containing any keyword in their name: `find`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
+> The search is case insensitive, the order of the keywords does not matter, only the name is searched,
 and persons matching at least one keyword will be returned (i.e. `OR` search).
 
 Examples: 
 * `find John`<br>
-  Returns `John Doe` but not `john`
+  Returns `John Doe` and 'john`
 * `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+  Returns Any person having names `Betsy`, `Tim`, `John`, 'bEtsy', 'TIM', ...
```

Comment:
Good job updating the user guide :) 

Just to nitpick, this symbol: ` and this symbol: ' are different symbols :P See what you typed for `john`